chainer/functions/activation/log_softmax.py

```diff
@@ -87,17 +87,16 @@ def backward(self, x, gy):
 
 
 def log_softmax(x, use_cudnn=True):
-    """Channelwise log-softmax function.
+    """Channel-wise log-softmax function.
 
     This function computes its logarithm of softmax along the second axis. Let
-    :math:`x = (x_1, x_2, \\dots, x_d)^{\\top}` be the d dimensional index
-    array and :math:`f(x_1, \\dots, x_d)` be the corresponding input array.
-    For each index :math:`x` in the input array, it computes the logarithm
-    of the probability :math:`\log p(x)` defined as
+    :math:`x = (x_1, x_2, \\dots, x_D)^{\\top}` be the D dimensional input
+    array.
+    For each input array :math:`x`, it computes the logarithm of the function
+    :math:`f(x)` defined as
```

Comment:
Comparing the description with input argument of this function, I felt that `x` in the document corresponds to `x` in the argument at first sight because it uses same variable name. But it is not true. Actually, `x` in the document is a narrowed version of `x` in the argument. How about changing the variable name of the two?